### DISCUSSION TITLE
Benchmarks / Comparison to Redpanda

### DISCUSSION THREAD

**Discussion by theduke** [fluvio-community/fluvio#2323] [General]
Hey there.

Out of curiosity: do you have any benchmarks that compare throughput / memory / cpu  usage etc with Kafka and other more optimized implementations like [Redpanda](https://github.com/redpanda-data/redpanda/)?

**Comment by sehz** [comment]
Hi, We are in process of building benchmark infrastructure.   The hard part is to make sure we are doing apples to apple comparison since there are so many knobs to configure.

**Reply by har07** [reply]
Is there any update on the benchmark?

**Reply by sehz** [reply]
Hi @har07, We have basic infrastructure in place. https://github.com/infinyon/fluvio/tree/master/crates/fluvio-benchmark and it runs as part of CI.   We are planning to add measurement for basic use cases.  Loop in @drc-infinyon
